extension-config.json

```diff
@@ -23,7 +23,7 @@
     "boldApiBaseUrl": {
       "type": "admin",
       "destination": "backend",
-      "default": "https://api-sandbox.bolt.com/",
+      "default": "",
```

Comment:
This key should be removed entirely so it would fail do deploy shops when not assigned. Otherwise it would just save empty strings as config values and then fail later on live.